lib/perl/Genome/Site/TGI/Extension/Sys/Lock.pm

```diff
@@ -41,7 +41,8 @@ if ($nessy_server) {
     );
     Genome::Sys::Lock->add_backend('site', $nessy_site_backend);
 
-    UR::Context->process->add_observer(
+    UR::Observer->register_callback(
+        subject => UR::Context->process,
```

Comment:
genome/genome#753 fixed this call.  `register_callback` does not support `subject`; `subject_id` and `subject_class_name` must be supplied.